qa-tests-backend/src/test/groovy/NetworkBaselineTest.groovy

```diff
@@ -275,58 +286,57 @@ class NetworkBaselineTest extends BaseSpecification {
         )
         validateBaseline(postLockClientBaseline, beforeDeferredCreate, justAfterDeferredCreate,
             [])
-    }
 
-    @Unroll
-    // TODO: ROX-11126
-    @Ignore
-    @Category(NetworkBaseline)
-    def "Verify user get for non-existent baseline"() {
         when:
-        "Create initial set of deployments, wait for baseline to populate"
-        def beforeDeploymentCreate = System.currentTimeSeconds()
-        batchCreate([USER_DEP, BASELINED_USER_CLIENT_DEP])
-        def justAfterDeploymentCreate = System.currentTimeSeconds()
+        "Verify user get for non-existent baseline"
+        beforeDeploymentCreate = System.currentTimeSeconds()
+        batchCreate([USER_DEP])
+        justAfterDeploymentCreate = System.currentTimeSeconds()
 
-        def serverDeploymentID = USER_DEP.deploymentUid
+        serverDeploymentID = USER_DEP.deploymentUid
```

Comment:
Now that both tests are merged, shouldn't we use a different variable name for this deployment? Maybe `userDeploymentID`? When reading the assertions below I was a bit confused because I thought `serverDeploymentID` was still referencing `SERVER_DEP` deployment other than `USER_DEP`.

Reply:
I will address that.  A case of being in a hurry and probably a little lazy.